master/internal/command/manager.go

```diff
@@ -42,8 +42,8 @@ func respondBadRequest(ctx *actor.Context, err error) {
 // - template: The configuration template name.
 // - user_files: The files to run with the command.
 // - data: Additional data for a command.
-func parseCommandRequest(
-	c echo.Context,
+func parseCommandRequestWithUser(
```

Comment:
other suggestions?

Reply:
that function has 3 other references in the code. If I change the interface to it, it would also need us to import our internal context/DetContext in those modules. seems simpler to allow a different version of the function? if we could overload them we wouldn't need a different fn name.

Reply:
```
internal/command/command_manager.go
internal/command/shell_manager.go
internal/command/notebook_manager.go
```
wdy think

Reply:
up to you, I dont have strong feelings. personally I'd just remove the one for echo stuff, but that's me.